validation/string.go

```diff
@@ -91,3 +91,36 @@ func (v *EndsWithValidator) MessagePlaceholders(_ *Context) []string {
 func EndsWith(suffix ...string) *EndsWithValidator {
 	return &EndsWithValidator{Suffix: suffix}
 }
+
+//------------------------------
+
+// DoesntStartWithValidator the field under validation must be a string not starting
+// with any of the specified prefixes.
+type DoesntStartWithValidator struct {
+	BaseValidator
+	Prefix []string
+}
+
+// Validate checks the field under validation satisfies this validator's criteria.
+func (v *DoesntStartWithValidator) Validate(ctx *Context) bool {
+	val, ok := ctx.Value.(string)
+	return !ok || !lo.ContainsBy(v.Prefix, func(prefix string) bool {
```

Comment:
To keep it consistent with `starts_with` and `ends_with`, the validator should return `false` if the value is not a `string`.